#! /bin/sh /usr/share/dpatch/dpatch-run
## 80_crmf.dpatch by  <glandium@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Put the crmf library before the NSS libraries

@DPATCH@

diff --git a/configure.in b/configure.in
index ef57009..f92b6e4 100644
--- a/configure.in
+++ b/configure.in
@@ -3746,7 +3746,7 @@ if test -n "$_USE_SYSTEM_NSS"; then
 fi
 
 if test -n "$MOZ_NATIVE_NSS"; then
-   NSS_LIBS="$NSS_LIBS -lcrmf"
+   NSS_LIBS=`echo $NSS_LIBS | sed 's/-l/-lcrmf &/'`
 else
    NSS_CFLAGS='-I$(DIST)/public/nss'
    NSS_DEP_LIBS='\\\
